project/AppDependencies.scala

```diff
@@ -7,7 +7,7 @@ object AppDependencies {
   val compile = Seq(
     ws,
     "uk.gov.hmrc"             %% "bootstrap-frontend-play-28" % "5.16.0",
-    "uk.gov.hmrc"             %% "play-frontend-hmrc"         % "1.26.0-play-28",
+    "uk.gov.hmrc"             %% "play-frontend-hmrc"         % "3.4.0-play-28",
```

Comment:
That's quite a bump. I guess everything still appears to hand together?

Reply:
Yep, all acceptance tests are running fine, and I've had a run through manually just to make sure nothing ugly is happening that the ATs wouldn't pick up.